### PR TITLE
Update the installation docs for CocoaPods to make the steps clearer.

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

File: docs/install-and-run.md [hunk 1]
```diff
@@ -54,72 +55,69 @@ For EarlGrey, we highly recommend [CocoaPods](https://cocoapods.org/) as the bes
   target TEST_TARGET do
     project PROJECT_NAME
 
-    use_frameworks!
-    inherit! :search_paths
+    use_frameworks! // Required for Swift Test Targets only
```
Comment (wuhao5) [inline docs/install-and-run.md]: I believe you can only do `# comments`, not `// comments` in ruby

File: docs/install-and-run.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ For EarlGrey, we highly recommend [CocoaPods](https://cocoapods.org/) as the bes
   the plus **(+)** sign, and then select the target from the dropdown menu. Select the **Shared** option and
   have the **Container** set to the app under test.
 
-    <img src="images/image01.png" width="500">
+      <img src="images/image01.png" width="500">
```
Comment (wuhao5) [inline docs/install-and-run.md]: is this extra two spaces intended?

File: docs/install-and-run.md (hunk 1 above)
Comment (tirodkar) [inline docs/install-and-run.md]: I'll change it. Thanks.

File: docs/install-and-run.md (hunk 2 above)
Comment (tirodkar) [inline docs/install-and-run.md]: It's needed to align it with the other  tags.

**Comment by tirodkar** [comment]
Done

File: docs/install-and-run.md [hunk 3]
```diff
@@ -54,72 +55,69 @@ For EarlGrey, we highly recommend [CocoaPods](https://cocoapods.org/) as the bes
...
-EarlGrey supports Carthage via the [EarlGrey gem.](https://rubygems.org/gems/earlgrey)
+EarlGrey supports Carthage via [the EarlGrey gem.](https://rubygems.org/gems/earlgrey)
 
 #### Step 1: Set up a test target for Carthage
 
-Follow [Step 1 from `CocoaPods installation`](#step-1-set-up-a-test-target) detailed above.
+See [Step 1 from the `CocoaPods installation`](#step-1-set-up-a-test-target) detailed above.
 
-#### Step 2: Configure Carthage
+#### Step 2: Configure carthage
```
Comment (sid-github) [inline docs/install-and-run.md]: Carthage

Comment (tirodkar) [inline docs/install-and-run.md]: Done.

**Pull request merged**